### PR TITLE
EVG-2155: skip tasks with distabled/missing projects

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

File: scheduler/task_finder.go [hunk 1]
```diff
@@ -76,11 +119,22 @@ func AlternateTaskFinder() ([]task.Task, error) {
 
 	runnabletasks := []task.Task{}
 	for _, t := range undispatchedTasks {
+		ref, ok := projectRefCache[t.Project]
+		if !ok {
```
Comment (bsamek) [inline scheduler/task_finder.go]: Log in these like the legacy finder.

File: scheduler/task_finder.go [hunk 2]
```diff
@@ -135,13 +195,42 @@ func ParallelTaskFinder() ([]task.Task, error) {
...
```
Comment (bsamek) [inline scheduler/task_finder.go]: Log.

File: scheduler/task_finder.go [hunk 3]
```diff
@@ -23,18 +25,54 @@ func LegacyFindRunnableTasks() ([]task.Task, error) {
...
+				"runner":  RunnerName,
+				"message": "could not find project for task",
+				"outcome": "skipping",
+				"task":    task.Id,
+				"project": task.Project,
+			})
+			continue
+		}
+
+		if !ref.Enabled {
```
Comment (bsamek) [inline scheduler/task_finder.go]: Add a test for this?

File: scheduler/task_finder.go (hunk 1 above)
Comment (tychoish) [reply scheduler/task_finder.go]: done

File: scheduler/task_finder.go (hunk 2 above)
Comment (tychoish) [reply scheduler/task_finder.go]: done.

File: scheduler/task_finder.go (hunk 3 above)
Comment (tychoish) [reply scheduler/task_finder.go]: I'm adding more data to the test fixtures that will exercise these parts of the code.

**Pull request merged**